ts/components/screens/help/HelpButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+import I18n from "../../../i18n";
+import ButtonDefaultOpacity, { ButtonSizes } from "../../ButtonDefaultOpacity";
+import IconFont from "../../ui/IconFont";
+
+type Props = {
+  onShowHelp: () => void;
+};
+
+const HelpButton: FC<Props> = ({ onShowHelp }) => (
+  <ButtonDefaultOpacity
+    onPress={onShowHelp}
+    style={ButtonSizes.minTouchableSquare}
```

Comment:
Instead of increase the button size could it be a better approach to extend the button hit area?
https://github.com/pagopa/io-app/blob/0e30730f595b2b041b667fc88cf45fd123ffd51f/ts/components/core/accessibility.ts#L7
https://reactnative.dev/docs/touchablewithoutfeedback#hitslop

Reply:
Before this PR the button was already `40` + `8` of increased hitSlop computed through `calculateSlop`, didn't really see any change in behavior